commit.go

```diff
@@ -249,8 +255,7 @@ func (b *Builder) Commit(dest types.ImageReference, options CommitOptions) error
 	}
 	if exporting {
 		// Copy everything.
-		// TODO: add credsContext
-		err = cp.Image(policyContext, dest, src, getCopyOptions(options.ReportWriter, nil, nil))
+		err = cp.Image(policyContext, dest, src, getCopyOptions(options.ReportWriter, nil, options.SystemContext))
```

Comment:
Question for @nalind .  cp.Image() does dest than src.  getCopyOptions does src then dest.  Can we change cp.Image or should I change getCopyOptions for the ordering?  I'm happy to do either, just wondering if changing cp.Image makes you twitch too much.

Reply:
TBH, I'm not at all bothered if they stay as they are.